samples/tests/ComponentExamples/ComponentExamples.test.ts

```diff
@@ -0,0 +1,63 @@
+import { test, expect } from '@playwright/test';
+import { encodeQueryData, createChatThreadAndUsers } from '../common/utils';
+import * as path from 'path';
+import dotenv from 'dotenv';
+
+dotenv.config({ path: path.join(__dirname, '..', '.env') });
+
+const SERVER_URL = 'http://localhost:8080';
+const PARTICIPANTS = ['Jack'];
+
+const dataUiId = (id: string): string => `[data-ui-id="${id}"]`;
+
+const dataUiStatus = (status: string): string => `[data-ui-status="${status}"]`;
+
+test.describe('Component Examples Test', () => {
+  test.beforeEach(async ({ page }) => {
+    // Go to the starting url before each test.
+    const user = await createChatThreadAndUsers(PARTICIPANTS);
```

Comment:
this is creating a chtthread and users, is it also provisioning a call? The tests reference calling:
![image](https://user-images.githubusercontent.com/2684369/141211194-3caf0a95-47b1-4abf-9335-4ec133b41bd7.png)
If so update this to something that indicates both chat details and call details are being created

Reply:
If it does, in our other tests we have to create a new call GUID every test or we get the tests hanging trying to rejoin the same call (issue is known to SDK team)